docs/changelog.mdx

```diff
@@ -4,6 +4,27 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.29
+
+_October 9th, 2024_
+
+**New Features**
+- Additional game and environment metadata is now captured during gameplay recording (e.g. Unity version, active input devices, etc.)
+- Bot Sequences can now be paused
+- Recordings are now saved directly within projects as a "Latest Recording" that can be copied and edited
+- Segments can now be seen and played directly from the in-game overlay
+- An `-rgsequencepath` argument can be used to start Unity and run a desired Bot Sequence
```

Comment:
Do we expect these changelogs to be self explanatory? If so I would specify what should be the type of the argument (path to the sequence?)

Reply:
In the future / past we link directly to the related docs. Since we are still in the process of catching up on the docs, we don't have a link to this yet, but in the future yes we should provide a way to learn more if possible.